.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  rok8s-scripts: fairwinds/rok8s-scripts@9.2.0
+  rok8s-scripts: fairwinds/rok8s-scripts@11.0.1
```

Comment:
If you want, you can pin to `@11` or `@11.0` if you want to automatically get updates. For e2e stuff I like `@11`